gbpservice/nfp/core/log.py

```diff
@@ -0,0 +1,123 @@
+from oslo_log import log as oslo_logging
+
+import inspect
+
+
+class NfpLogMeta(object):
+
+    def __init__(self, **kwargs):
+        self.meta_id = kwargs.get('meta_id', '')
+        self.event = kwargs.get('event', '')
```

Comment:
Are we sending event in meta information? This will be present in all logs if this is passed right?